docs/mesos/README.md

```diff
@@ -46,3 +46,17 @@ a.)  We've bundled Mesos with netmodules into a convenient RPM - follow the [Mes
 b.) Already have mesos installed? Netmodules can be compiled onto an existing mesos deployment so long as all the mesos source files are present (therefore, unfortunately at this time, netmodules can not be added to mesos when installed via the mesosphere rpm releases). Follow the [manual netmodules compilation guide](ManualInstallNetmodules.md) to install it.
 
 [![Analytics](https://ga-beacon.appspot.com/UA-52125893-3/calico-docker/docs/mesos/README.md?pixel)](https://github.com/igrigorik/ga-beacon)
+
+## 5. Launching Tasks
+Calico is compatible with all frameworks which use the new NetworkInfo protobuf when launching tasks. Marathon has introduced limited support for this. For an early peek, use `djosborne/marathon:ip_per_task`:
+```
+$ docker run \
+-e MARATHON_MASTER=zk://<ZOOKEEPER-IP>:2181/mesos \
+-e MARATHON_ZK=zk://<ZOOKEEPER-IP>:2181/marathon \
+-p 8080:8080 \
+djosborne/marathon:ip_per_task
+```
+The Marathon UI has does not yet include a field for specifiying NetworkInfo, so we'll use the command line to send Marathon an App's JSON instead:
+```
+$ curl -X PUT -H "Content-Type: application/json" http://localhost:8080/v2/groups/calico-apps  -d '{"id":"/calico-apps", "apps": [{"id": "hello-world-1", "cmd": "ifconfig", "cpus": 0.1, "mem": 64.0, "ipAddress": {"groups": ["netgroup-1"]}}]}'
```

Comment:
Can you pretty-print this blob of JSON and explicitly explain the new `ipAddress` field?